src/components/Button/ButtonWrapper.tsx

```diff
@@ -3,11 +3,11 @@ import Icon from "../Icon"
 import { ButtonBase, ButtonInset } from "./components"
 import { ThemeContext } from "styled-components"
 import { parseCustomColor, parseColorPreset } from "../../common/_utils"
-import { ButtonWrapperProps } from "./types"
+import { ButtonWrapperInterface, ButtonWrapperProps } from "./types"
 
 const DEFAULT_COLOR_THEME = "neutral"
 
-const ButtonWrapper: React.FC<ButtonWrapperProps> = ({
+const ButtonWrapper: ButtonWrapperInterface<ButtonWrapperProps> = ({
```

Comment:
Can you also change Button props to extend for HTMLButton type like it has been done for Input component?

Reply:
`ButtonHTMLAttributes` have type property itself, which will require to change our API, which require to change Buttons API wherever knit-ui is using it. This task is not much of priority, we will do it later.